Make-a-pull-request.md

```diff
@@ -193,4 +236,26 @@ git push origin --delete new-branch-name
 
 ## Step 8: Celebrate
 
-Congratulations, you have contributed to Oppia! To learn some more advanced ways to use PRs at Oppia, see our [[guide to common PR workflows|Common-pull-request-workflows]].
+Congratulations, you have successfully contributed to Oppia! 🎉
+
+### Tips for Managing Multiple PRs
```

Comment:
Hmm I don't think this is really a sub-section of step 8. I'd make it a level 2 heading instead

Reply:
Sounds reasonable. I'll make it a level 2 heading and add it to the Table of Contents.